uint8 servo_id
uint16 position
